app/queue/queue.py

```diff
@@ -126,7 +126,7 @@ def add_host(host_data):
                 extra={"host": {i: output_host[i] for i in output_host if i not in ("facts", "system_profile")}},
             )
             payload_tracker_processing_ctx.inventory_id = output_host["id"]
-            return (output_host, add_result)
+            return (output_host, insights_id, add_result)
```

Comment:
If we retrieve the Host ID above, we need to pass it out here too.

```suggestion
            return output_host, host_id, insights_id, add_result
```